src/main/java/frc/robot/commands/Robot2022PanicCMD.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.logging.RobotLogManager;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.subsystems.Indexer2022;
+import frc.robot.subsystems.LlamaNeck2022;
+import frc.robot.subsystems.Spitter2022;
+import org.apache.logging.log4j.Logger;
+
+public class Robot2022PanicCMD extends CommandBase {
```

Comment:
This is a currently unimplemented Command to quickly cancel all commands on the shooter system and drivetrain.

Reply:
Where do you intend to use this? The name "Panic" has a strong meaning in computer systems. Maybe "CancelAll" would be a better name. But I still don't fully understand the intent.

Reply:
It might not be needed, but mainly it is intended to quickly cancel all running commands, this could be used if something went wrong but we don't want to disable and re enable the robot.